package.json

```diff
@@ -35,5 +38,8 @@
     "prettier": "2.8.3",
     "tsconfig-paths": "4.1.2",
     "typescript": "4.9.4"
+  },
+  "peerDependencies": {
+    "class-validator": "0.13.2"
```

Comment:
precisa instalar como peerDependencies? To achando que poderia ficar em dependencies normal em. Deu algum bug ao instalar?

Reply:
É para dar uma ênfase de que deve ser essa versão para rodar sem problemas e que também é uma dependência do TypeORM